app/controllers/onboardings_controller.rb

```diff
@@ -0,0 +1,98 @@
+class OnboardingsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :check_suspended, only: %i[notifications]
+  before_action :set_cache_control_headers, only: %i[show tags]
+  before_action :set_no_cache_header, only: %i[update]
+  after_action :verify_authorized, only: %i[update checkbox]
+
+  TAG_ONBOARDING_ATTRIBUTES = %i[id name taggings_count].freeze
+  ALLOWED_USER_PARAMS = %i[last_onboarding_page username].freeze
+  ALLOWED_CHECKBOX_PARAMS = %i[checked_code_of_conduct checked_terms_and_conditions].freeze
+  ALLOWED_NOTIFICATION_PARAMS = %i[email_newsletter email_digest_periodic].freeze
+
+  def show
+    set_surrogate_key_header "onboarding-slideshow"
```

Comment:
I'm trying to level up my understanding of this surrogate keys syntax today; is the goal here to efficiently purge onboarding-slideshow cache? Or just to set the header to allow for that purging to happen elsewhere? Not a blocking question, I'm just learning!